plugins/protocolbuffers/java/v24.0/buf.plugin.yaml

```diff
@@ -0,0 +1,19 @@
+version: v1
+name: buf.build/protocolbuffers/java
+plugin_version: v24.0
+source_url: https://github.com/protocolbuffers/protobuf
+description: Base types for Java. Generates message and enum types.
+output_languages:
+  - java
+spdx_license_id: BSD-3-Clause
+license_url: https://github.com/protocolbuffers/protobuf/blob/v24.0/LICENSE
+registry:
+  maven:
+    deps:
+      - com.google.protobuf:protobuf-java:3.24.0
+    additional_runtimes:
+      - name: lite
+        deps:
+          - com.google.protobuf:protobuf-javalite:3.24.0
+          - build.buf:protobuf-javalite:3.24.0
```

Comment:
Before merging this PR, verify this is published.

Reply:
Going to back out the java changes to avoid blocking the rest of the releases, and deal with that separately.